resources/todomvc/big-dom-generator/utils/buildComplex.js

```diff
@@ -0,0 +1,73 @@
+const fs = require("fs");
+const { JSDOM } = require("jsdom");
```

Comment:
`jsdom` needs to be in `dependencies ` instead of `devDependencies` so that the script works in projects. I believe that's the only one.

Reply:
Actually I take that back, it's not necessary, I just needed to run `npm i` first in `big-dom-generator`.

That said I think it's still more correct to have `jsdom` in the `dependencies` list, but it's optional then.